build.sbt

```diff
@@ -12,17 +12,20 @@ val akkaHttpVersion = "10.1.3"
 val circeVersion = "0.9.3"
 val doobieVersion = "0.5.2"
 
+libraryDependencies += jdbc
+libraryDependencies += "org.postgresql" % "postgresql" % "9.4.1208"
```

Comment:
Are you sure that explicit adding of this deps is necessary, since doobie is still used under the hood ?